integration-tests/utils/integration-utils.js

```diff
@@ -37,7 +37,7 @@ const getClickablePath = (path, text, textContains, isDescendant = false) => {
   if (text) {
     return `${base}[normalize-space(.)="${text}" or @title="${text}" or @aria-label="${text}" or @aria-labelledby=//*[normalize-space(.)="${text}"]/@id]`
   } else if (textContains) {
-    return `${base}[contains(normalize-space(.),"${textContains}") or contains(@title,"${textContains}") or contains(@aria-label,"${textContains}") or @aria-labelledby=//*[contains(normalize-space(.),"${textContains}")]/@id]`
+    return `${base}[contains(normalize-space(.),"${textContains}") or contains(@title,"${textContains}") or contains(@alt,"${textContains}") or contains(@aria-label,"${textContains}") or @aria-labelledby=//*[contains(normalize-space(.),"${textContains}")]/@id]`
```

Comment:
I think we leave out the alt text since we are doing the separate img method.

Reply:
That piece is needed actually because `image` function still calls `getClickablePath` and the resulting XPath doesn't match the alt text without `contains(@alt,"${textContains}")`.

Reply:
I'm still feeling uneasy about this because images are not generally clickable. I'd like to see what some other folks things-- @petesantos , @zarsky-broad ?

Reply:
I'm still feeling uneasy about this because images are not generally clickable. I'd like to see what some other folks things-- @petesantos , @zarsky-broad ?

Reply:
I also think that if you are going to make this change, there is really no point in having a separate `image` method.

If we keep the separate image method, then I think it should not rely on `getClickablePath`.

Reply:
And one more comment, you only changed the "textContains" path to handle the alt text.

Reply:
Just as a functionality question, is the alt text something we could potentially request on specific components/elements or is it solely for accessibility right now?

Reply:
alt text is specific to images, and it is required for screen readers to interpret images.

All images ​must​ include an ​alt​ attribute. There are some cases when an
empty ​alt​ attribute (​alt=""​) is desirable (if the image is purely decorative and there is already text on the page that describes what the image is displaying, for example if there is a button with both a word and an image). However, the ​alt​ attribute must never be omitted. An empty alt attribute has special significance to screen reader users. When a
screen reader encounters an image with an empty alt attribute, it remains silent. It essentially skips over the image as if it weren't there. If an author fails to include an alt attribute at all, the screen reader will attempt a series of fallback behaviors in an effort to give the user any helpful information it can about the image. The last of these fallbacks
is to read the path to the image. This might actually be helpful if the path to an image that links to the "About us" page is www.example.com/aboutus.gif. However, that is rarely the case. It is often something like www.example.com/theme/thYJ3JIeKD79_45_2H.jpg, which isn't very helpful at all.

Reply:
I strongly agree with @cahrens - just because it works doesn't mean it's a good idea. The more we follow semantic best practices, the less we'll have to build (and maintain!) custom tooling for our workarounds.

Reply:
I think at this point it comes down to UX. Do we want "buttons" with only images on them, or should there be text on the button as well as the image (in which case the image is decorative and should have an empty alt attribute so that the information is not read out twice via the screen reader)? In the case of these "buttons", there is also the UX issue that they do not have a visible hover state. I really expected that this would have all been sorted out through the process of user testing, as the controls just feel really weird, but given that it hasn't been sorted out, I can create a ticket for these issues.

Reply:
Thank you for the very thorough reviews, @cahrens. Please do create tickets as needed. The team's goal _is_ to iterate and improve Analysis Tab UX 🤞🏼 

Addressed XPath stuff in [66bb9d7](https://github.com/DataBiosphere/terra-ui/pull/2865/commits/66bb9d75112ddd1ac43c9a74ab9d55f1685fad17)

Reply:
This is better with the hover color. Thanks!